modules/monitoring-supported-remote-write-authentication-settings.adoc

```diff
@@ -0,0 +1,314 @@
+
+|===
+
+== Config map location for authentication settings
+The following shows the location of the authentication configuration in the `ConfigMap` object for default platform monitoring.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com" <1>
+        <endpoint_authentication_details> <2>
+----
+<1> The URL of the remote write endpoint.
+<2> The required configuration details for the authentication method for the endpoint.
+Currently supported authentication methods are Amazon Web Services (AWS) Signature Version 4, bearer token, basic authentication, OAuth 2.0, and TLS client. 
+
+[NOTE]
+====
+If you want to configure remote write for the Prometheus instance that monitors user-defined projects, edit the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace.
+====
+
+== Example remote write authentication settings
+
+The following samples show different authentication settings you can use to connect to a remote write endpoint.
+Each sample also shows how to configure a corresponding `Secret` object that contains authentication credentials and other relevant settings.
+Each sample configures authentication for use with default platform monitoring in the `openshift-monitoring` namespace.
+
+.Sample YAML for AWS Signature Version 4 authentication
+
+The following shows the settings for a `sigv4` secret named `sigv4-credentials` in the `openshift-monitoring` namespace.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sigv4-credentials
+  namespace: openshift-monitoring
+stringData:
+  access: <AWS_access_key> <1>
+  secret: <AWS_secret_key> <2>
+type: Opaque
+----
+<1> The AWS API access key.
+<2> The AWS API secret key.
+
+The following shows sample AWS Signature Version 4 remote write authentication settings that use a `Secret` object named `sigv4-credentials` in the `openshift-monitoring` namespace:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://authorization.example.com/api/write"
+        sigv4:
+          region: <AWS_region> <1>
+          accessKey:
+            name: sigv4-credentials <2>
+            key: access <3>
+          secretKey:
+            name: sigv4-credentials <2>
+            key: secret <3>
+          profile: <AWS_profile_name> <4>
+          roleArn: <AWS_role_arn> <5>
+----
+<1> The AWS region.
+<2> The name of the `Secret` object containing the AWS API access credentials.
+<3> The AWS API access credentials contained in the specified `Secret` object.
```

Comment:
To be consistent with the rest of the examples I would expect each field to have its own footnote.
```suggestion
<3> The key in the specified `Secret` object containing the AWS access key.
<4> The key in the specified `Secret` object containing the AWS secret key.
```

Reply:
Added the extra calloout/footnote and edited per your suggestion as follows:

```
<3> The key that contains the AWS API access key in the specified `Secret` object.
<4> The key that contains teh AWS API secret key in the specified `Secret` object.
<5> The name of the AWS profile that is being used to authenticate.
<6> The unique identifier for the Amazon Resource Name (ARN) assigned to your role.
```